test/ICEDdemo.py

```diff
@@ -707,81 +711,87 @@ def mainIced(Crows, Columns, uFactor):
 
     Parameters: NULL
 
     Returns: NULL
     """
     print("ICEDdemo.py starts running.")
     timeStart = time.time()
+    
     CGRAsizes = [6]  # the mapped CGRA size = 6
-    unrollFactors = [1, 2]   # unrolling factor = 1, 2
-    nameCvsBaseline = []    
-    cvsPath = ""
+    unrollFactors = [2]   # unrolling factor = 1, 2
+    nameCsvBaseline = []    
+    csvPath = ""
     for CGRAsize in CGRAsizes:
         for unrollFactor in unrollFactors:      
-            nameCvsBaseline.append(mainBaseline(CGRAsize, CGRAsize, unrollFactor, False)) 
+            nameCsvBaseline.append(mainBaseline(CGRAsize, CGRAsize, unrollFactor, False)) 
     print("Generating Table I: Target workloads and streaming applications")
-    cvsPath = "./example/exampleTable.cvs"
-    showTableI(cvsPath, nameCvsBaseline)
+    csvPath = "./example/exampleTable.csv"
+    showTableI(csvPath, nameCsvBaseline)
 
-    print("Replace testBenchs to generate Fig. 9, 10, 11.")
+    # print("Replace testBenchs to generate Fig. 9, 10, 11.")
     # fig091011GenerationKernel()
-    CGRAsizes = [6]  # the mapped CGRA size = 6
-    unrollFactors = [1, 2]   # unrolling factor = 1, 2
-    nameCvsBaseline = []    
-    nameCvsPertile = []   
-    nameCvsIced = []    
-    figPath = ""
-    for CGRAsize in CGRAsizes:
-        for unrollFactor in unrollFactors:      
-            nameCvsBaseline.append(mainBaseline(CGRAsize, CGRAsize, unrollFactor))
-            nameCvsPertile.append(mainPertile(CGRAsize, CGRAsize, unrollFactor))
-            nameCvsIced.append(mainIced(CGRAsize, CGRAsize, unrollFactor))  
-    print("Generating Fig. 9: Average utilization of tiles across different kernels")
-    figPath = "./example/exampleFig9.png"
-    showFig9(figPath, nameCvsBaseline, nameCvsPertile, nameCvsIced)   
-
-    print("Generating Fig. 10: Average DVFS level across different kernels")
-    figPath = "./example/exampleFig10.png"
-    showFig10(figPath, nameCvsBaseline, nameCvsPertile, nameCvsIced)   
-
-    print("Generating Fig. 11: Evaluation of energy-efficiency")
-    figPath = "./example/exampleFig11.png"
-    showFig11(figPath, nameCvsBaseline, nameCvsPertile, nameCvsIced)
-
-    print("Replace testBenchs to generate Fig. 12.") 
+    # CGRAsizes = [6]  # the mapped CGRA size = 6
+    # unrollFactors = [1, 2]   # unrolling factor = 1, 2
+    # nameCsvBaseline = []    
+    # nameCsvPertile = []   
+    # nameCsvIced = []    
+    # figPath = ""
+    # for CGRAsize in CGRAsizes:
+    #     for unrollFactor in unrollFactors:      
+    #         nameCsvBaseline.append(mainBaseline(CGRAsize, CGRAsize, unrollFactor))
+    #         nameCsvPertile.append(mainPertile(CGRAsize, CGRAsize, unrollFactor))
+    #         nameCsvIced.append(mainIced(CGRAsize, CGRAsize, unrollFactor))  
+    # print("Generating Fig. 9: Average utilization of tiles across different kernels")
+    # figPath = "./example/exampleFig9.png"
+    # showFig9(figPath, nameCsvBaseline, nameCsvPertile, nameCsvIced)   
```

Comment:
How could the showFig9() be performed if this is commented out?

Reply:
Sorry for the mistake.